src/app/pages/LiveRadioPage/index.test.jsx

```diff
@@ -29,8 +29,12 @@ describe('Radio Page Main', () => {
 
     await matchSnapshotAsync(
       <ToggleContextProvider
-        service="amharic"
-        origin="https://www.test.bbc.com"
+        remoteToggles={{
+          service: 'amharic',
+          ads: {
```

Comment:
It might be good to use a testToggle here instead of `ads` across the board

Reply:
Removed this from a bunch of places and just let it default to `null`. It wasn't needed in the context of any of these tests